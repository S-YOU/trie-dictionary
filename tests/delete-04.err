nodes: 2
nodes: 4
nodes: 4
nodes: 4
