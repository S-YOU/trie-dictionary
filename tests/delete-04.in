insert xwtcso
insert xpap
insert x
delete 2
