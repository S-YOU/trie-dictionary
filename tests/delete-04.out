word number: 0
word number: 1
word number: 2
deleted: 2
